CHANGELOG.md

```diff
@@ -17,6 +17,8 @@
 - [#91](https://github.com/clojure-emacs/clojure-ts-mode/pull/91): Introduce `clojure-ts-cycle-keyword-string`.
 - [#92](https://github.com/clojure-emacs/clojure-ts-mode/pull/92): Add commands to convert between collections types.
 - [#93](https://github.com/clojure-emacs/clojure-ts-mode/pull/93): Introduce `clojure-ts-add-arity`.
+- Fix an issue where `clojure-ts-align` would hang when called within an
```

Comment:
That's not needed, as the functionality hasn't been released yet.

Reply:
Removed.